ods_ci/tests/Tests/400__ods_dashboard/440__model_registry/440__model_registry.robot

```diff
@@ -0,0 +1,107 @@
+${JUPYTER_NOTEBOOK_FILEPATH}=        ${MODELREGISTRY_BASE_FOLDER}/${JUPYTER_NOTEBOOK}
+${DC_S3_TYPE}=                       Object storage
+
+
+*** Test Cases ***
+Verify Model Registry Integration With Jupyter Notebook
+    [Documentation]    Verifies the Integartion of Model Registry operator with Jupyter Notebook
+    [Tags]    OpenDataHub     RHOAIENG-4501
+    Create Workbench    workbench_title=${WORKBENCH_TITLE}    workbench_description=Registry test
+    ...                 prj_title=${PRJ_TITLE}    image_name=Minimal Python  deployment_size=Small
+    ...                 storage=Persistent   pv_existent=${NONE}
+    ...                 pv_name=${NONE}  pv_description=${NONE}  pv_size=${NONE}
+    Workbench Should Be Listed      workbench_title=registry-wb
+    Open Data Science Project Details Page       project_title=${PRJ_TITLE}
+    ${workbenches}=    Create List    ${WORKBENCH_TITLE}
+    Create S3 Data Connection    project_title=${PRJ_TITLE}    dc_name=${DC_S3_NAME}
+    ...            aws_access_key=${S3.AWS_ACCESS_KEY_ID}    aws_secret_access=${S3.AWS_SECRET_ACCESS_KEY}
+    ...            aws_bucket_name=${aws_bucket}    connected_workbench=${workbenches}
+    Data Connection Should Be Listed    name=${DC_S3_NAME}    type=${DC_S3_TYPE}    connected_workbench=${workbenches}
+    Wait Until Workbench Is Started     workbench_title=registry-wb
+    Wait For Model Registry Containers To Be Ready
+    Upload File In The Workbench     filepath=${SAMPLE_ONNX_MODEL}    workbench_title=${WORKBENCH_TITLE}
+    ...         workbench_namespace=${PRJ_TITLE}
+    Upload File In The Workbench     filepath=${JUPYTER_NOTEBOOK_FILEPATH}    workbench_title=${WORKBENCH_TITLE}
+    ...         workbench_namespace=${PRJ_TITLE}
+    Launch And Access Workbench    workbench_title=${WORKBENCH_TITLE}
+    ...    username=${TEST_USER_3.USERNAME}     password=${TEST_USER_3.PASSWORD}
+    ...    auth_type=${TEST_USER_3.AUTH_TYPE}
+    Jupyter Notebook Should Run Successfully     ${JUPYTER_NOTEBOOK}
+
+*** Keywords ***
+Prepare Model Registry Test Setup
+    [Documentation]    Suite setup steps for testing Model Registry.
+    Set Library Search Order    SeleniumLibrary
+    #Skip If Component Is Not Enabled    modelregistry
+    RHOSi Setup
+    Launch Dashboard    ${TEST_USER.USERNAME}    ${TEST_USER.PASSWORD}    ${TEST_USER.AUTH_TYPE}
+    ...    ${ODH_DASHBOARD_URL}    ${BROWSER.NAME}    ${BROWSER.OPTIONS}
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    Apply Db Config Samples    namespace=${PRJ_TITLE}
+    Fetch CA Certificate If RHODS Is Self-Managed
+
+Apply Db Config Samples
+    [Documentation]    Applying the db config samples from https://github.com/opendatahub-io/model-registry-operator
+    [Arguments]    ${namespace}
+    ${rc}    ${out}=    Run And Return Rc And Output
+    ...    oc apply -k ${MODEL_REGISTRY_DB_SAMPLES}/mysql -n ${namespace}
```

Comment:
I'm a bit concerned about the need to replicate Manifest in `/samples` from the Operator repo to locally; as eventually these will get out of sync. Isn't it just better to do something ~like:
```
oc apply -k "github.com/opendatahub-io/model-registry-operator/config/samples/mysql?ref=main" -n ${namespace}
```
?

That way, we don't have to "remember" to keep files in sync

Reply:
Thanks @tarilabs, It was my first approach and had done this in a similar way by cloning the repo. The only concern QE person had was we won't be able to use github in a Disconnected Cluster and the tests won't run there. So we would have to have a workaround for disconnected cluster.